src/test/mochitest/browser_dbg-sourcemaps.js

```diff
@@ -78,13 +78,14 @@ add_task(async function() {
   assertPausedLocation(dbg);
 
   await stepIn(dbg);
-  // await waitForMappedScopes(dbg);
+  await waitForSelectedSource(dbg, "times2.js")
```

Comment:
Is this somehow related to the change to the way the pause state is managed? If not, it should be a separate pull request.